modules/tiles/src/tileset/helpers/bounding-volume.ts

```diff
@@ -51,13 +55,29 @@ export function createBoundingVolume(boundingVolumeHeader, transform, result) {
       [degrees(east), degrees(south), maxHeight],
       scratchSouthEast
     );
-    const centerInCartesian = new Vector3().addVectors(northWest, southEast).multiplyScalar(0.5);
-    const radius = new Vector3().subVectors(northWest, southEast).len() / 2.0;
+    const centerInCartesian = new Vector3().addVectors(northWest, southEast).multiplyByScalar(0.5);
+    Ellipsoid.WGS84.cartesianToCartographic(centerInCartesian, scratchCenter);
 
-    // TODO improve region boundingVolume
-    // for now, create a sphere as the boundingVolume instead of box
-    return createSphere(
-      [centerInCartesian[0], centerInCartesian[1], centerInCartesian[2], radius],
+    Ellipsoid.WGS84.cartographicToCartesian(
```

Comment:
Code is long enough that creating a helper function `convertRegionToOBB` could make sense?